src/components/ec-radio-btn/ec-radio-btn.spec.ts

```diff
@@ -0,0 +1,301 @@
+        options: [
+          { value: 'y', label: 'Testing label' },
+        ],
+      });
+
+      expect(wrapper.findByDataTest('ec-radio-btn__radio-description').exists()).toBe(false);
+    });
+  });
+
+  describe('#slots', () => {
+    it('#errorMessage - should render the error message slot if passed', () => {
+      const wrapper = mountRadioBtn(
+        {},
+        {
+          slots: {
+            'error-message': 'Test error message slot',
+          },
+        },
+      );
+
+      expect(wrapper.findByDataTest('ec-radio-btn__error-text').exists()).toBe(true);
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('#error-message - should render error-message slot instead of error-message prop when both are passed', () => {
+      const wrapper = mountRadioBtn(
+        {
+          errorMessage: 'Error message coming from props',
+        },
+        {
+          slots: {
+            'error-message': 'Error message coming from slot',
+          },
+        },
+      );
+
+      expect(wrapper.findByDataTest('ec-radio-btn__error-text').exists()).toBe(true);
+      expect(wrapper.element).toMatchSnapshot();
+    });
+  });
+
+  describe('@events', () => {
+    it('@update:modelValue - should be emitted when input is clicked', async () => {
+      const wrapper = mountRadioBtn({
+        options: [
+          { value: 'y', label: 'Yes' },
+          { value: 'n', label: 'No' },
+        ],
+      });
+
+      await wrapper.findByDataTest('ec-radio-btn-0').trigger('click');
+      expect(wrapper.emitted('update:modelValue')?.length).toBe(1);
+    });
+  });
+
+  describe('attrs', () => {
+    it('should render with custom attributes', () => {
+      const wrapper = mountRadioBtn({}, {
+        attrs: {
+          'data-test': 'my-data-test',
+          class: 'my-class',
+          id: 'test-id',
+        },
+      });
+      expect(wrapper.element).toMatchSnapshot();
+    });
+  });
+
+  describe('v-model', () => {
+    it('should render the radio button and toggle v-model value when input is clicked', async () => {
+      const wrapper = mountRadioBtn({
+        label: 'Test label prop',
+        options: [
+          { value: 'y', label: 'Yes' },
+          { value: 'n', label: 'No' },
+        ],
+        modelValue: 'y',
+      });
+
+      await wrapper.findByDataTest('ec-radio-btn-1').trigger('click');
+      expect(wrapper.emitted()['update:modelValue'][0]).toEqual(['n']);
+    });
+
+    it('should not change the value of v-model if disabled is enabled and user clicks the radio button', async () => {
+      const wrapper = mountRadioBtn({
+        label: 'Test label prop',
+        options: [
+          { value: 'y', label: 'Yes' },
+          { value: 'n', label: 'No' },
+        ],
+        modelValue: 'y',
+        isDisabled: true,
+      });
+
+      await wrapper.findByDataTest('ec-radio-btn-1').trigger('click');
+      expect(wrapper.emitted()['update:modelValue']).toEqual(undefined);
+    });
+  });
```

Comment:
we can remove this, component doesn't manage it's modelValue. It just gets value from props and emits.

Reply:
done, thank you